EmotiBit.cpp

```
@@ -1574,6 +1598,9 @@ void EmotiBit::parseIncomingControlPackets(String &controlPackets, uint16_t &pac
 			else if (header.typeTag.equals(EmotiBitPacket::TypeTag::MODE_WIRELESS_OFF)) {
 				setPowerMode(EmotiBit::PowerMode::WIRELESS_OFF);
 			}
+			else if (header.typeTag.equals(EmotiBitPacket::TypeTag::MODE_BLUETOOTH)) {
```

Clang (14.0.6)

[note] +1, nesting level increased to 3

(clang)